websocket/logs.go

```diff
@@ -134,7 +134,7 @@ func (h *handlerWithLogs) Receiver() hwebsocket.Receiver {
 				WithTag(logs.SessionIDTag, h.sessionID).
 				WithTag("session_uuid", h.sessionUUID).
 				WithTag(logs.ParticipantIDTag, h.participantID).
-				Error(errors.New("receiving message failed").Wrap(err))
+				Warn(errors.New("receiving message failed").Wrap(err))
```

Comment:
Does this happen often and it's a false alarm or why do we wanna silence it?

Reply:
this is error from the socket level, just thought there is nothing we can do about this. 

reverted the log level.

Reply:
Is this one of those places we could get "eof" or "connection closed by peer"? If yes, I agree it's not actionable and should probably be a warning indeed.

Reply:
EOF & Connection closed by peer is excluded though.
```
&& !errors.Is(err, io.EOF) && !errors.Is(err, net.ErrClosed)
```